src/erlcass.erl

```diff
@@ -368,6 +415,21 @@ receive_response(Tag) ->
         {error, timeout}
     end.
 
+receive_paged_response(Tag) ->
+    receive
+        {paged_execute_statement_result, Tag, {ok, Columns, Rows}} ->
```

Comment:
Instead of paged_execute_statement_result an paged_execute_statement_result_has_more why don't you return from nif the result into the final format like: 

{paged_execute_statement_result, Tag, {ok, Columns, Rows, HasMore}} ?